backend/app/adapter/graphql/resolver/change.go

```diff
@@ -0,0 +1,30 @@
+package resolver
+
+import (
+	"github.com/short-d/short/app/adapter/graphql/scalar"
+	"github.com/short-d/short/app/entity"
+)
+
+type Change struct {
+	change entity.Change
+}
+
+func (c Change) Id() string {
+	return c.change.ID
+}
+
+func (c Change) Title() *string {
+	return &c.change.Title
+}
+
+func (c Change) SummaryMarkdown() *string {
+	return &c.change.SummaryMarkdown
+}
+
+func (c Change) ReleasedAt() scalar.Time {
+	return scalar.Time{Time: *c.change.ReleasedAt}
+}
+
+func newChange(change entity.Change) *Change {
```

Comment:
```suggestion
func newChange(change entity.Change) Change {
```

Reply:
`*` means pointer. We use it when some of the methods forces us to operate on pointer, such as something like:
```
func (c *Change) MutateThisStruct() *string
```